services/tenant-ui/frontend/src/components/verifier/PresentationDetails.vue

```diff
@@ -53,6 +68,21 @@
         <!-- requested_attribute using 'name' string w/o restrictions -> revealed self-attested values -->
         <!-- requested_predicates -> unrevealed attributes -->
       </div>
+      <hr />
+      <!-- Identifiers -->
+      <Accordion>
+        <AccordionTab
+          v-bind:header="`Identifier_${index + 1}`"
+          v-for="(item, index) in props.presentation.acapy.presentation_exchange
```

Comment:
Would a lint complain about no ':key' on this one?

Reply:
yes it did, fixed.